app/helpers/ui_helper.rb

```diff
@@ -6,14 +6,14 @@ def grouped_options_for_assigning_case(volunteer)
         CasaCase
           .not_assigned(@volunteer.casa_org).active
           .uniq { |casa_case| casa_case.case_number }
-          .map { |casa_case| [casa_case.case_number, casa_case.id] }
+          .map { |casa_case| ["#{casa_case.case_number} - #{render AssignedVolunteersComponent.new(casa_case, current_user)}".html_safe, casa_case.id] }
```

Comment:
This is just producing text, right? If so, a method in the Casa Case model would be enough.

Reply:
UI helper is a good place for UI specific logic.